src/components/EventSection/EventSection.jsx

```diff
@@ -0,0 +1,138 @@
+    </>
+);
+
+const DriftTaxiCardContent = () => (
+    <>
+        Doświadcz prawdziwych motoryzacyjnych emocji! Widowiskowe pokazy driftu,
+        przejażdżki na fotelu pasażera i adrenalina na najwyższym poziomie! W
+        specjalnej strefie expo czekają na Ciebie najnowsze trendy motoryzacyjne
+        oraz renomowane marki z Polski i Europy.
+    </>
+);
+
+const EventSection = () => {
+    const handleLinkClick = (label) => {
+        if (window.gtag) {
+            window.gtag("event", "event_section_link_click", {
+                event_category: "EventSection",
+                event_label: label,
+            });
+        }
+    };
+
+    useEffect(() => {
+        const script = document.createElement("script");
+        script.type = "application/ld+json";
+        script.text = JSON.stringify({
+            "@context": "https://schema.org",
+            "@type": "Event",
+            name: "Street Meeting Poland 2025",
+            startDate: "2025-05-31T12:00:00+02:00",
+            endDate: "2025-05-31T23:00:00+02:00",
+            location: {
+                "@type": "Place",
+                name: "Polsat Plus Arena",
+                address: {
+                    "@type": "PostalAddress",
+                    streetAddress: "Pokoleń Lechii Gdańsk 1",
+                    addressLocality: "Gdańsk",
+                    postalCode: "80-560",
+                    addressCountry: "PL",
+                },
+            },
+            description:
+                "Przyjedź i odwiedź jedyne takie wydarzenie motoryzacyjne w Polsce.",
+            offers: {
+                "@type": "Offer",
+                url: "https://bkb.pl/151648-edcca",
+                priceCurrency: "PLN",
+                availability: "https://schema.org/InStock",
+            },
+        });
+        document.head.appendChild(script);
+
+        return () => {
+            document.head.removeChild(script);
+        };
+    }, []);
+
+    return (
+        <section id="event" className={styles.event}>
+            <div className={styles.container}>
+                <h2
+                    className={styles.title}
+                    aria-label="Informacje o wydarzeniu Street Meeting Poland 2025"
+                >
+                    Event
+                </h2>
+                <p className={styles.description}>
+                    Przyjedź i odwiedź jedyne takie wydarzenie w Polsce.
+                </p>
+                <div className="row row-cols-1 row-cols-md-2 row-cols-xl-3 g-4">
+                    <Card
+                        imgSrc="/photos/10.webp"
+                        imgAlt="Samochody na murawie Polsat Plus Arena, Gdańsk, a na przodzie składane krzesełka z napisem Street Meeting"
+                        title="Bilety"
+                        link="https://bkb.pl/151648-edcca"
+                        imgHeight="300px"
+                    >
+                        <TicketCardContent onLinkClick={handleLinkClick} />
```

Comment:
I think this could be part of the CardComponent, for example as Content. Then you can pass content (text or HTML?) to the card and remove all those strange components